Sources/Stripe/API/Routes/EphemeralKeyRoutes.swift

```diff
@@ -11,7 +11,7 @@ import Vapor
 public protocol EphemeralKeyRoutes {
     associatedtype EK: EphemeralKey
     
-    func create(customer: String) throws -> Future<EK>
+    func create(customer: String, apiVersion: String?) throws -> Future<EK>
```

Comment:
Since this is a bit of a breaking chance, my suggestion is to give `apiVersion: String?` a default value of `nil` since you're already checking if it's optional or not. 
```swift
func create(customer: String, apiVersion: String? = nil) throws -> Future<EK>
```

Reply:
@Sorix ^^

Reply:
You can't use default arguments in protocols as far as I know.